src/main/java/Contact.java

```diff
@@ -0,0 +1,34 @@
+
+public class Contact {
```

Comment:
Code is bang on bro, might just need an auto-format on this class, we can try and get something set up so you can do this a bit more intuitively too

Reply:
When we can get on a call for 5